resources/definitions/dagoma_discoeasy200.def.json

```diff
@@ -29,32 +29,47 @@
         },
         "machine_head_with_fans_polygon": {
             "default_value": [
-                [17, 70],
-                [17, -40],
-                [-17, -40],
-                [17, 70]
+                [-17, -70],
+                [-17, 40],
+                [17, 40],
+                [17, -70]
             ]
         },
         "gantry_height": {
             "default_value": 10
         },
         "machine_start_gcode": {
-            "default_value": ";Gcode by Cura\nG90\nM106 S250\nG28 X Y\nG1 X50\nM109 S180\nG28\nM104 S{material_print_temperature_layer_0}\nG29\nM107\nG1 X100 Y20 F3000\nG1 Z0.5\nM109 S{material_print_temperature_layer_0}\nM82\nG92 E0\nG1 F200 E10\nG92 E0\nG1 Z3\nG1 F6000\n"
+            "default_value": ";Gcode by Cura\nG90\nM106 S255\nG28 X Y\nG1 X50\nM109 R90\nG28\nM104 S{material_print_temperature_layer_0}\nG29\nM107\nG1 X100 Y20 F3000\nG1 Z0.5\nM109 S{material_print_temperature_layer_0}\nM82\nG92 E0\nG1 F200 E10\nG92 E0\nG1 Z3\nG1 F6000\n"
         },
         "machine_end_gcode": {
             "default_value": "\nM104 S0\nM106 S255\nM140 S0\nG91\nG1 E-1 F300\nG1 Z+3 F3000\nG90\nG28 X Y\nM107\nM84\n"
         },
+        "material_diameter": {
```

Comment:
`material_diameter` is now in the extruder definitions.

Reply:
Fixed in ed1e47d